src/cpp/shared/adventure/hero.cpp

```diff
@@ -312,15 +312,15 @@ int hero::CalcMobility() {
       }
       auto res = ScriptCallbackResult<int>("OnCalcMobility", deepbind<hero*>(this), points);
       if(res.has_value())
-        points = res.value();
+        points = max(1, res.value());
```

Comment:
Should be max(0, ...), right?

Reply:
It crashes because of integer division by 0. It's mobility, not remaining mobility